app/dashboard/src/providers/DriveProvider.tsx

```diff
@@ -53,7 +54,7 @@ export default function DriveProvider(props: ProjectsProviderProps) {
       setCanDownload: (canDownload) => {
         set({ canDownload })
       },
-      selectedKeys: new Set(),
+      selectedKeys: EMPTY_SET,
```

Comment:
It means that we're using the same `Set()` object everywhere where we reference `EMPTY_SET`. Is it what we want?

Reply:
yeah, this is intentional. it's a premature optimization both to try and avoid some unnecessary re-renders (which, to be fair, should seldom happen) and to avoid some GC pressure in JS (again only very minimal improvements there)

Reply:
But it also means that we might have a _not_ empty set because somewhere mutated it. Not sure these optimizations worth it.